lib/CleantalkSP/SpbctWP/FSWatcher/View/View.php

```diff
@@ -96,13 +92,67 @@ public static function getExtendedTabDescription(Phrases $phrases)
         return $phrases->getExtendedTabDescription();
     }
 
+    protected static function renderSelectorOptions($dates)
+    {
+        if (Controller::$debug) {
+            Logger::log($dates);
+        }
+
+        $html = '';
+        foreach ($dates as $date) {
+            $formated_date = date('M d Y H:i:s', $date);
```

Comment:
I suggest that CleanTalk standard data format is Y-m-d H:i:s

Reply:
Done https://github.com/CleanTalk/security-malware-firewall/commit/eb0dbdefa652e66b6bd972c9160eefafa1b338be